src/Metacello-Core/MetacelloProjectSpecForLoad.class.st

```diff
@@ -85,10 +85,18 @@ MetacelloProjectSpecForLoad >> performCurrentVersionTestAgainst: vrsn operator:
 MetacelloProjectSpecForLoad >> performLoad [
   | displayString spec |
   spec := self projectSpec.
-  displayString := 'Project: ' , spec name.
+
+displayString := 'Project: ' , spec name.
+	
   spec versionString ~~ nil
     ifTrue: [ displayString := displayString , ' ' , spec versionString ].
-  MetacelloNotification signal: displayString.
+  	MetacelloNotification signal: displayString.
+
+	spec isLocked ifTrue: [ 
+		MetacelloNotification signal: 'Project is locked, skipping.'.
```

Comment:
we should use custom notification class or at least an exception tag here

Reply:
I use what Metacello uses to do logging, the intention is not other than the information to arrive to the transcript :P